packages/overlay/src/topLayerOverTransforms.js

```diff
@@ -10,4 +10,95 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
-export * from '@swc-uxp-internal/overlay/src/topLayerOverTransforms.js';
+"use strict";
+import {
+  getContainingBlock,
+  getWindow,
+  isContainingBlock
+} from "@floating-ui/utils/dom";
+import { VirtualTrigger } from "./VirtualTrigger.dev.js";
+export const topLayerOverTransforms = () => ({
+  name: "topLayer",
+  async fn(middlewareArguments) {
+    const {
+      x,
+      y,
+      elements: { reference, floating }
+    } = middlewareArguments;
+    //Making it by default since we don't have a way to check if the floating is on top layer in UXP.
+    let onTopLayer = true;
```

Comment:
is it going to be ever any scenario that onTopLayer can be false? or we want it be true always?

Reply:
There could be but we don't any way to check that in UXP hence keeping it true.